library/testing/TestSDKDynamicLightIntensity.cxx

```diff
@@ -0,0 +1,47 @@
+#include <engine.h>
+#include <loader.h>
+#include <options.h>
+#include <window.h>
+
+#include "TestSDKHelpers.h"
+
+int TestSDKDynamicLightIntensity(int argc, char* argv[])
```

Comment:
You could have added one line in `cmake/testing.cmake` it would have been easier, but it's fine the way you did it.  
@mwestphal we should add a section in the documentation explaining how to make test.

Reply:
Yeah I couldn't figure it out so I went with TestSDKDynamicBackgroundColor as a model.

In the end it worked out best this way because the test covers changing the intensity dynamically (ie. the test will fail if `this->OriginalLightIntensities[light] = originalIntensity;` is removed, which would not be a problem when `vtkF3DRenderer::SetLightIntensity` is only called once)

Reply:
Very good point @Meakk !

It would be nice to also add a --light-intensity test in cmake/testing.cmake though.

Reply:
I barely managed to write and run this one test by poking around and making educated guesses, I still don't know how to properly run the whole test suite.
You guys need to document how to 1. run the existing tests, and 2. write new ones if you don't want new contributors to give up before their feature is considered good enough

Reply:
Ok, that's fair. We are currently rewriting the whole documentation but it is taking some time.

Thanks a lot for your patience and tenacity. I'll add another test once we merge this.

Reply:
Here is the doc, still a WIP, but should make things clearer:

https://github.com/mwestphal/f3d/blob/jekyll_doc/doc/dev/TESTING.md